orca-core/src/test/groovy/com/netflix/spinnaker/orca/pipeline/util/PackageInfoSpec.groovy

```diff
@@ -192,6 +192,41 @@ class PackageInfoSpec extends Specification {
 
   }
 
+  @Unroll
+  def "Find the right package (NUPKG), don't be dependant on artifact order"() {
```

Comment:
Since this new test case appears to be the same as the one just above, with the exception of `packageType` and the fileName extensions, we could just add a `packageType` column to the `where:` table and add the additional cases to the table. Please add some `.deb` cases as well since your hands are already in the code.